Content.Shared/SimpleStation14/CCVar/CCVars.cs

```diff
@@ -16,4 +16,17 @@ public sealed class SimpleStationCCVars
     /// </summary>
     public static readonly CVarDef<List<string>> AnnouncerBlacklist =
         CVarDef.Create("game.announcer.blacklist", new List<string>(), CVar.SERVERONLY);
+
+
+    /// <summary>
+    ///     Whether or not respawning is enabled.
+    /// </summary>
+    public static readonly CVarDef<bool> RespawnEnabled =
+        CVarDef.Create("respawn.enabled", true, CVar.SERVER | CVar.REPLICATED);
+
+    /// <summary>
+    ///     How long the player has to wait in seconds after death before respawning.
+    /// </summary>
+    public static readonly CVarDef<float> RespawnTime =
+        CVarDef.Create("respawn.time", 600.0f, CVar.SERVER | CVar.REPLICATED);
```

Comment:
```suggestion
    /// <summary>
    ///     Whether or not respawning is enabled.
    /// </summary>
    public static readonly CVarDef<bool> RespawnEnabled =
        CVarDef.Create("game.respawn_enabled", true, CVar.SERVER | CVar.REPLICATED);

    /// <summary>
    ///     How long the player has to wait in seconds after death before respawning.
    /// </summary>
    public static readonly CVarDef<int> RespawnTime =
        CVarDef.Create("game.respawn_time", 600, CVar.SERVER | CVar.REPLICATED);
```
This is probably fine, don't you think? I'm not against making a new category, just don't want to do it unnecessarily.

Also, no reason to wait six hundred and a half seconds >:P
Maybe consider doing this in minutes though, as that's a lot more readable. Then it can be a float, for 0.5 minutes or what have you.
`TimeSpan.FromMinutes(<CCVar>);`